model/host/db.go

```diff
@@ -577,6 +578,10 @@ func ByIP(ip string) db.Q {
 	return db.Query(bson.M{IPKey: ip})
 }
 
+func ByIPv4(ipv4 string) db.Q {
+	return db.Query(bson.M{IPv4Key: ipv4})
```

Comment:
Let's remember to add an index for this.

Reply:
need to add {"ipv4_address":1} index
should ask in ops after deployed!